doc: add information about FILE_SUFFIX (#14766)
nrfconnect/sdk-nrf: applications/zigbee_weather_station/README.rst

.. _zigbee_weather_station_app_build_types:

Zigbee weather station configurations
=====================================

The Zigbee weather station application does not use a single :file:`prj.conf` file.

> The application includes different configuration files, which are located in the :file:`configuration/thingy53_nrf5340_cpuapp` directory.
> Before you start testing, you can select one of them using the :makevar:`FILE_SUFFIX` variable.

Reviewer: ```suggestion
The application includes different configurations, with files for each configuration located in the :file:`configuration/thingy53_nrf5340_cpuapp` directory.
Before you start testing, you can select one of the configurations using the :makevar:`FILE_SUFFIX` variable.
```
Maybe change like this? Makes the focus more on the configurations instead of the files.
